src/components/navigation/Header.tsx

```diff
@@ -75,9 +86,24 @@ function Header(header:NavProps): JSX.Element {
       skipToContentLabel="Skip to main content"
       title={t('site_name')}
       titleAriaLabel={t('navigation.title_aria_label')}
+      className={classNames(styles.navigation, styles.zover)}
     >
+      <Navigation.Row>
+        {getNavi(menu)}
+      </Navigation.Row>
       <Navigation.Actions>
-        <Navigation.Row variant='inline' key='languages'>
+        <Navigation.Search searchLabel="Search" searchPlaceholder="Search page" />
+        <Navigation.User
```

Comment:
Should these "Search" strings be translated?

Reply:
Placeholder, we don't have search yet so this is just for documentation now.